Appraisals

```diff
@@ -1,7 +1,7 @@
 ruby_version = Gem::Version.new(RUBY_VERSION + '')
 
 shared_dependencies = proc do
-  gem 'rspec-rails', '>= 3.2.0', '< 4'
+  gem 'rspec-rails', '>= 3.3.0', '< 4'
```

Comment:
Mind rebasing and extracting this change as well as the changes to Gemfile and Gemfile.lock to a separate commit?